prefect_email/credentials.py

```diff
@@ -90,11 +92,11 @@ class EmailServerCredentials(Block):
     _block_type_name = "Email Server Credentials"
     _logo_url = "https://images.ctfassets.net/gm98wzqotmnx/3PcxFuO9XUqs7wU9MiUBMg/ca740e27815d15528373aced667f58b9/email__1_.png?h=250"  # noqa
 
-    username: str
-    password: SecretStr
-    smtp_server: Optional[Union[str, SMTPServer]] = SMTPServer.GMAIL
-    smtp_type: Optional[Union[str, SMTPType]] = SMTPType.SSL
-    smtp_port: Optional[int] = None
+    username: str = None
```

Comment:
I think we still need Optional for the ones that default to None.
```suggestion
    username: Optional[str] = None
```